components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml

```diff
@@ -2397,9 +2397,8 @@ return PhenoTips;
         onSuccess: function (response) {
           this.setContent(this.buildTree(this._getDataFromResponse(response)));
           this.__crtRoot = id;
-          if (this.container.content) {
-            Event.fire(document, 'xwiki:dom:updated', {'elements' : [this.container.content]});
-          }
+          if (this.container.contentContainer != null)
```

Comment:
But it causes a regression:
- in edit mode, open the "Clinical symptoms" section
- expand one of the categories
- click on the `i` info button next to one of the phenotypes
- click on "Browse related terms..."
- click on the `i` button next to the displayed phenotype or its parents, but not its children
- nothing happens

The standard field for most containers (instances of `PhenoTips.widgets.ModualPopup) is `content`, `contentContainer` is custom defined for one specific type of element, the branches shown when expanding a predefined phenotype.

In other words, this change is wrong.